components/legacy_team_sidebar/legacy_team_sidebar_controller.tsx

```diff
@@ -168,11 +179,11 @@ export default class LegacyTeamSidebar extends React.PureComponent {
         const destinationIndex = result.destination.index;
 
         // Positioning the dropped Team button
-        const popElement = (list, idx) => {
+        const popElement = (list: string | any[], idx: number) => {
```

Comment:
why the 'any' here? what else can be in the list except for string?

Reply:
fixed it below